### PR TITLE
Script to auto-generate docs issues from release notes file

### PR DESCRIPTION
**Pull request by jseldess**
@knz and/or @benesch, this is ready for a review.

### CODE REVIEW

File: scripts/issues-from-release-notes.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import argparse
```
Comment (benesch) [inline scripts/issues-from-release-notes.py]: If you add a line like

```
#!/usr/bin/env python3
```

(a "shebang"), then run

```
chmod +x scripts/issues-from-release-notes.py
```

then users can directly run scripts/issues-from-release-notes.py without needing to think about which Python version to use.

File: scripts/issues-from-release-notes.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+args = parser.parse_args()
+
+release_notes = args.release_notes
+milestone = args.milestone
+
+# Map milestones to their internal IDs.
+if milestone == "2.1":
+    milestone = 8
+if milestone == "2.O.x":
+    milestone = 10
```
Comment (benesch) [inline scripts/issues-from-release-notes.py]: One thing to consider is asking GitHub for milestone title -> id mapping: https://developer.github.com/v3/issues/milestones/

File: scripts/issues-from-release-notes.py (hunk 1 above)
Comment (jseldess) [reply scripts/issues-from-release-notes.py]: Done. Thanks!

File: scripts/issues-from-release-notes.py (hunk 2 above)
Comment (jseldess) [reply scripts/issues-from-release-notes.py]: This is a great idea, but I'll think I'll take it up in a follow-up PR.

**Comment by jseldess** [comment]
TFTR, @benesch!

**Pull request merged**